Palmer_McKay/app_base.htm

```diff
@@ -0,0 +1,95 @@
+<%inherit file="/homepage/templates/app_base.htm" />
+<%! from catalog import models as cmod %>
+
+<%block name="left">
+    <div id="category_list" class="col-sm-2 ">
+        <ul class="seo-list">
+            <a id="all" class="categories" href="/catalog/index"><li>All Products</li></a>
+            % for cat in cmod.Category.objects.order_by('name'):
+                <a class="categories" href="/catalog/index/${cat.id}"><li>${ cat.name }</li></a>
+            % endfor
+        </ul>
+    </div>
+</%block>
+
+<%block name="right">
+  <div class="col-md-2">
+   <h2 class = "right_list_title">Recently Viewed</h2>
+   <ol class = "right_list">
+   % for product in request.last_five[1:6]:
+     <a href="/catalog/detail/${product.id}">
+       <li>
+         <p class="right_pane_name">${product.name}</p>
```

Comment:
I think product name underneath the image might look better, but not positive.

Reply:
That is worth looking into